core/actor/src/main/scala/net/liftweb/actor/LAFuture.scala

```diff
@@ -295,13 +298,13 @@ object LAFuture {
     }
   }
 
-  private def executeWithObservers(scheduler: LAScheduler, f: () => Unit) {
+  private def executeWithObservers(scheduler: LAScheduler, f: () => Unit, context:ContextFn) {
```

Comment:
No, I tried that per the Codacy recommendation. The companion class can't see it.